### PR TITLE
github: added PULL_REQEUEST_TEMPLATE for PR maintenance

### PR DESCRIPTION
**Pull request by leemgs**
This commit is to improve PR template (e.g., summary, goal, description, reproduction, and so on) that can be submitted by contributers to evolve the FauxPilot software.

Signed-off-by: Geunsik Lim <leemgs@gmail.com>
Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

### CODE REVIEW

File: .github/pull-request-template.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- [ ] Checked if README should be updated
+
+
+## 1. General Description
+* {your contribution}
+* {links to design docs, confluence pages, etc}
+*
+
+## 2. Related JIRA Tasks & GitHub Issues
+* https://code.sec.samsung.net/jira/browse/{JIRA}
```
Comment (fdegier) [inline .github/pull-request-template.md:15]: Since this is not related to Samsung I suggest to remove it.
```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+
+---
+# [Template] PR Description
```
Comment (fdegier) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: Why are there 2 PR templates? The second one seems more appropriate for FauxPilot. What do you think?

**Review by fdegier — CHANGES_REQUESTED** [review]

File: .github/pull-request-template.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- [ ] The code coverage is higher than 80% 
+- [ ] Checked if README should be updated
+
+
+## 1. General Description
+* {your contribution}
+* {links to design docs, confluence pages, etc}
+*
+
+## 2. Related JIRA Tasks & GitHub Issues
```
[the remark this answers is not in the document]
Comment (leemgs) [reply .github/pull-request-template.md:14]: > 

Thanks. Fixed.

File: .github/pull-request-template.md (hunk 1 above)
Comment (leemgs) [reply .github/pull-request-template.md:15]: > Since this is not related to Samsung I suggest to remove it.

Nice Catch. Updated.

File: .github/pull-request-template.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+* https://code.sec.samsung.net/jira/browse/{JIRA}
+* Fixes #{GitHub Issue Number}
+
+## 3. How to write a commit title for changelog
+Commit title must be started with one of the eleven types listed below.
+e.g., build, chore, ci, docs, feat, fix, perf, refactor, revert, style, and test.
+* https://github.sec.samsung.net/RS-SA/code-review-bot/blob/develop/Documentation/changelog-commit-rule.md
```
[the remark this answers is not in the document]
Comment (leemgs) [reply .github/pull-request-template.md:21]: > Url is not publicly accessible.

Fixed. Thanks.

**Comment by leemgs** [comment]
To reviewers

**Changelog**
 * Version 2
   - Applied comments of reviewer(s)
   - Updated the sections of a PR writing guide for readability.

 * Version 1
   - Added Pull Request template


@moyix , @fdegier, I have updated the existing PR according to the @fdegier 's comments. Could you review the latest version?

**Comment by leemgs** [comment]
## Screenshot after a latest version. 

image

**Comment by leemgs** [comment]
This PR is ready to be merged. Pleast review the latest version.

**Review by fdegier — APPROVED** [review]

**Comment by fdegier** [comment]
Thank you @leemgs 🙏
